File: backend/src/main/java/com/abel/eventos/infrastructure/adapter/in/web/dto/VentaResponse.java
package com.abel.eventos.infrastructure.adapter.in.web.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.Instant;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class VentaResponse {

    private Long id;
    private Long eventoId;
    private Instant fechaVenta;
    private BigDecimal precioVenta;
    private Boolean resultado;
    private String descripcion;
    private List<AsientoResponse> asientos;
}

File: mobile/composeApp/src/commonMain/kotlin/org/abel/mobile/data/model/VentaResponse.kt
package org.abel.mobile.data.model

import kotlinx.serialization.Serializable

@Serializable
data class VentaResponse(
    val id: Long? = null,
    val eventoId: Long? = null,
    val fechaVenta: String? = null,
    val precioVenta: Double? = null,
    val resultado: Boolean? = null,
    val descripcion: String? = null,
    val asientos: List<Asiento>? = null
)